# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsWindow.cpp
# Commit: 786823305560
# Full Hash: 786823305560b03316cb758207f76838388603af
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2020-05-05 03:49:49
# Regressor Bug: 1574746
# File Overlap Count: 1
# Description:
#   Bug 1574746 - Remove AllowWebRenderForThisWindow() r=nical
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D71440
# ==============================================================================

diff -r 7af7b4435997 -r 786823305560 widget/gtk/nsWindow.cpp
--- a/widget/gtk/nsWindow.cpp	Mon May 04 23:13:55 2020 +0000
+++ b/widget/gtk/nsWindow.cpp	Sun May 03 11:38:53 2020 +0000
@@ -4161,10 +4161,8 @@
       // gfxVars, used below.
       Unused << gfxPlatform::GetPlatform();
 
-      bool useWebRender =
-          gfx::gfxVars::UseWebRender() && AllowWebRenderForThisWindow();
-
       mIsAccelerated = ComputeShouldAccelerate();
+      bool useWebRender = gfx::gfxVars::UseWebRender() && mIsAccelerated;
       MOZ_ASSERT(mIsAccelerated | !useWebRender);
 
       if (mWindowType == eWindowType_toplevel) {